src/server/twemcache-http/build.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2019 Twitter, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::env;
+use std::path::Path;
+
+fn main() {
+    let flag = if cfg!(target_os = "linux") {
+        "OS_LINUX"
+    } else {
```

Comment:
might be better to check if target os matches for mac os and otherwise exit with error